Makefile

```diff
@@ -476,6 +478,38 @@ docker-iqe-vortex-tests:
 	$(MAKE) docker-reinitdb
 	./testing/run_vortex_api_tests.sh
 
+### Provider targets ###
+ocp_provider_from_yaml:
+#parameter validation
+ifndef cluster_id
+	$(error param cluster_id is not set)
+endif
+ifndef srf_yaml
+	$(error param srf_yaml is not set)
+endif
+ifndef ocp_name
+override ocp_name = $(cluster_id)
+endif
+	(command -v nise > /dev/null 2>&1) || (echo 'nise is not installed, please install nise.' && exit 1 )
+	mkdir -p testing/pvc_dir/insights_local
+	nise --ocp --ocp-cluster-id $(cluster_id) --insights-upload testing/pvc_dir/insights_local --static-report-file $(srf_yaml)
+	curl -d '{"name": "$(ocp_name)", "type": "OCP", "authentication": {"provider_resource_name": "$(cluster_id)"}}' -H "Content-Type: application/json" -X POST http://0.0.0.0:8000/api/cost-management/v1/providers/
+# These csv could be cleaned up by: https://github.com/project-koku/koku/issues/1602 is resolved.
```

Comment:
```suggestion
# These csv could be cleaned up when [https://github.com/project-koku/nise/issues/176](https://github.com/project-koku/nise/issues/176) is resolved.
```

Reply:
@maskarb Sweet! Thanks for showing me the suggestion feature in github. I have never seen it before, but it pretty neat. I am fore sure gonna start using this in the future for pr reviews.